lib/src/scope.dart

```diff
@@ -46,7 +46,7 @@ class Scope {
   /// Calls an isolate-scoped RPC named [method] with [params].
   ///
   /// This always adds the `isolateId` parameter to the RPC.
-  Future<Object> sendRequest(String method, [Map<String, Object> params]) async {
+  Future<dynamic> sendRequest(String method, [Map<String, Object> params]) async {
```

Comment:
No, I'm saying make the return value `Future<Map<String, dynamic>>` and cast in the body of this function. Note that to be runtime-strong-mode safe, you'll need to use [`new UnmodifiableMap.typed()`](https://www.dartdocs.org/documentation/collection/1.9.0/collection/DelegatingMap/typed.html).